README.md

```diff
@@ -10,19 +10,89 @@ Assuming it has been published to RubyGems, you can install this gem using:
 you@machine $ inspec plugin install inspec-vault
 ```
 
+## Loading Secrets into Vault
+
+Getting started with Vault is beyond the scope of this document, but you can download a recent version from https://www.vaultproject.io . Then start a Vault dev-mode server:
+
+```
+$ vault server -dev
+```
+
+And you can then store an input. Let's assume you want to store an input named `my_input` with the value 2, for the profile `my_profile`.
+
+```
+[cwolfe@lodi inspec-vault]$ vault kv put secret/inspec/my_profile my_input=2
+Key              Value
+---              -----
+created_time     2019-09-10T17:54:16.237055Z
+deletion_time    n/a
+destroyed        false
+version          1
+```
+
+With that value stored, Inspec will now be able to retrieve the value.
+
 ## What This Plugin Does
 
-No idea.
+Whenever profile code like this is encountered:
```

Comment:
It feels like half a sentence is missing. Was the sentence meant to continue after the profile code example (below, lines 39 - 44)?

Reply:
I've added transitional sentences before and after.